Octokit.Tests.Conventions/Exception/InterfaceNotFoundException.cs

```diff
@@ -8,18 +8,20 @@ public class InterfaceNotFoundException : Exception
         public InterfaceNotFoundException() { }
 
         public InterfaceNotFoundException(string type)
-            : base(CreateMessage(type)) { }
+            : base(CreateMessage(type))
+        { }
 
```

Comment:
I had a look into this, and it turns out the rule is working this way due to:

 - open brace always on a newline
 - close brace (where trivial) is moved on to the same line

If someone wants to fight me on this and put it back, they're more than welcome to poke around and see if they can fix this over on my CodeFormatter fork - it's `BraceNewLineRule` you want to look at...